### PR TITLE
Create or Update alias API supports must_exist parameter

### PR DESCRIPTION
**Pull request by gaobinlong**
### Description
In version 2.12.0, we enable the parameter must_exist for the Remove action in Create/Update aliases API, this PR documents the usage.

### Issues Resolved
Fixes https://github.com/opensearch-project/documentation-website/issues/6124

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
Thanks for adding this PR, @gaobinlong! Is it ready for a doc review?

File: _im-plugin/index-alias.md [hunk 1]
```diff
@@ -126,6 +126,23 @@ POST _aliases
 }
 ```
 
+In addition, `remove` action supports `must_exist` paramater, if it is `true` and the specified alias doesn't exist, an exception will be thrown, if it is `false`, nothing happens if the specified alias doesn't exist. The default value of `must_exist` is `null`, only if all of the specified aliases don't exist, an exception will be thrown.
```
Comment (vagimeli) [inline _im-plugin/index-alias.md:129]:
```suggestion
The `remove` action also supports the `must_exist` parameter. If the parameter is set to `true` and the specified alias does not exist, an exception is thrown. If the parameter is `false`, then no action is taken if the specified alias does not exist. The default value for `must_exist` is `null`. An exception will be thrown only if none of the specified aliases exist.
```

File: _im-plugin/index-alias.md [hunk 2]
```diff
@@ -247,4 +264,4 @@ For example, if `alias1` refers to `index-1` and `index-2`, you can run the foll
 DELETE index-1/_alias/alias1
 ```
 
-After you run the request above, `alias1` no longer refers to `index-1`, but still refers to `index-2`.
\ No newline at end of file
+After you run the request above, `alias1` no longer refers to `index-1`, but still refers to `index-2`.
```
Comment (vagimeli) [inline _im-plugin/index-alias.md:267]:
```suggestion
After running the previous request, `alias1` no longer refers to `index-1`, but still refers to `index-2`.
```

**Review by vagimeli — APPROVED** [review]
@gaobinlong Please see my doc review edits to make sure I've retained the intended meaning. I'll move this PR into editorial review following your review. Thanks, Melissa

**Comment by gaobinlong** [comment]
> @gaobinlong Please see my doc review edits to make sure I've retained the intended meaning. I'll move this PR into editorial review following your review. Thanks, Melissa

The suggested change looks good to me, I've changed that, thanks!

File: _im-plugin/index-alias.md [hunk 3]
```diff
@@ -76,17 +71,18 @@ To check if `alias1` refers to `index-1`, run one of the following commands:
 GET /_alias/alias1
 GET /index-1/_alias/alias1
 ```
+{% include copy-curl.html %}
 
-To get the mappings and settings information of the indexes that the alias references, run the following command:
+To get the indexes' mappings and settings information that the alias references, run the following command:
```
Comment (natebower) [inline _im-plugin/index-alias.md:76]:
```suggestion
To get the indexes' mappings and settings information referenced by the alias, run the following command:
```

File: _im-plugin/index-alias.md [hunk 4]
```diff
@@ -125,6 +120,27 @@ POST _aliases
   ]
 }
 ```
+{% include copy-curl.html %}
+
+The `remove` action also supports the `must_exist` parameter. If the parameter is set to `true` and the specified alias does not exist, an exception is thrown. If the parameter is set to `false`, then no action is taken if the specified alias does not exist. The default value for `must_exist` is `null`. An exception will be thrown only if none of the specified aliases exist. 
+
+The following POST request uses the `remove` action, with `must_exist` parameter set to `true`:
```
Comment (natebower) [inline _im-plugin/index-alias.md:127]:
```suggestion
The following POST request uses the `remove` action with the `must_exist` parameter set to `true`:
```

File: _im-plugin/index-alias.md [hunk 5]
```diff
@@ -178,10 +200,11 @@ HEAD /alias1/_alias/
 HEAD /_alias/alias1/
 HEAD index-1/_alias/alias1/
 ```
+{% include copy-curl.html %}
 
 ## Add aliases at index creation
 
-You can add an index to an alias as you create the index:
+You can add an index to an alias as you create the index like that shown in the following PUT request:
```
Comment (natebower) [inline _im-plugin/index-alias.md:207]:
```suggestion
You can add an index to an alias as you create the index, as shown in the following PUT request:
```

File: _im-plugin/index-alias.md [hunk 6]
```diff
@@ -191,12 +214,11 @@ PUT index-1
...
 }
 ```
+{% include copy-curl.html %}
 
 ## Create filtered aliases
 
-You can create a filtered alias to access a subset of documents or fields from the underlying indexes.
-
-This command adds only a specific timestamp field to `alias1`:
+You can create a filtered alias to access a subset of documents or fields from the underlying indexes. This command adds only a specific timestamp field to `alias1`. The following shows a POST request example:
```
Comment (natebower) [inline _im-plugin/index-alias.md:221]:
```suggestion
You can create a filtered alias to access a subset of documents or fields in the underlying indexes. This command adds only a specific timestamp field to `alias1`. The following shows an example POST request:
```

File: _im-plugin/index-alias.md [hunk 7]
```diff
@@ -76,17 +71,18 @@ To check if `alias1` refers to `index-1`, run one of the following commands:
...
 ```json
 GET alias1
 ```
+{% include copy-curl.html %}
 
 ## Add or remove indexes
 
-You can perform multiple actions in the same `_aliases` operation.
-For example, the following command removes `index-1` and adds `index-2` to `alias1`:
+You can perform multiple actions in the same `_aliases` operation. For example, the following command removes `index-1` and adds `index-2` to `alias1`:
```
Comment (natebower) [inline _im-plugin/index-alias.md:85]:
```suggestion
You can perform multiple actions using the same `_aliases` operation. For example, the following command removes `index-1` and adds `index-2` to `alias1`:
```

**Review by natebower — COMMENTED** [review]
@vagimeli Just a few changes. Thanks!

**Pull request merged**